system/Autoloader/FileLocatorCached.php

```diff
@@ -86,6 +81,12 @@ public function deleteCache(): void
         $this->cacheHandler->delete($this->cacheKey);
     }
 
+    /**
+     * Find the qualified name of a file according to
+     * the namespace of the first matched namespace path.
+     *
+     * @return false|string The qualified name or false if the path is not found
+     */
```

Comment:
Why do you need to add this doc comment?
It is already written in the `FileLocatorInterface`.